install_config/http_proxies.adoc

```diff
@@ -47,6 +47,37 @@ NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16 <1>
 ----
 <1> Supports host names and CIDRs. Must include the SDN and service IP ranges `10.1.0.0/16,172.30.0.0/16` by default.
 ====
++
+`*NO_PROXY*` accepts a comma-separated list of hosts:
++
+For master hosts::
+* Node hostname
+* Master IP or hostname
+* Service IP CIDR
+* Cluster IP CIDR
+For node hosts::
+* Master IP or hostname
+* Service CIDR
+* Cluster IP CIDR
+For the Docker service::
+* Registry service IP and hostname
++
+[NOTE]
```

Comment:
Opinion here, but having two note admonitions back to back always looks a bit wrong to my eyes (especially with the way they render in OpenShiftland).

I think the first NOTE could just be plain text, and leave the second one as an admonition.